src/cogniteClient.ts

```diff
@@ -289,14 +289,32 @@ export default class CogniteClient {
    * ```
    */
   public getMetadata = (value: any) => this.metadataMap.get(value);
+  public get = (path: string, options?: BaseRequestOptions) =>
+    this.doRawRequest('get', path, options);
+
+  public put = (path: string, options?: BaseRequestOptions) =>
+    this.doRawRequest('put', path, options);
+
+  public post = (path: string, options?: BaseRequestOptions) =>
+    this.doRawRequest('post', path, options);
+
+  public delete = (path: string, options?: BaseRequestOptions) =>
+    this.doRawRequest('delete', path, options);
 
   /**
    * Do plain get-requests using the client
    */
-  public get = (path: string, options?: BaseRequestOptions) =>
-    rawRequest(this.instance, { method: 'get', url: path, ...options }).then(
-      responseTransformer
-    );
+
```

Comment:
add `// @hidden` here in order to hide the function from the documentation

Reply:
Done